include/flox/core/nix-state.hh

```diff
@@ -139,15 +141,16 @@ public:
    *
    * Evaluator remains open for lifetime of object.
    */
-  nix::ref<nix::EvalState>
+    nix::ref<nix::EvalState>
   getState()
   {
     if ( this->state == nullptr )
       {
-        this->state
-          = std::make_shared<nix::EvalState>( std::list<std::string> {},
-                                              this->getStore(),
-                                              this->getStore() );
+        this->state = std::make_shared<nix::EvalState>(
+          nix::SearchPath()
```

Comment:
question nonblocking: why this change?

Reply:
upstream changed the API for this function call. (ref: https://github.com/NixOS/nix/pull/8579/files#diff-f118e4c6f6e02148b887fdf627352311fca5a3a4eadf0b4a9d9f348e0be464ffL500-L501)